agent/ecscni/plugin.go

```diff
@@ -31,7 +31,11 @@ import (
 	"github.com/pkg/errors"
 )
 
-const currentCNISpec = "0.3.1"
+const (
+	currentCNISpec = "0.3.1"
+	currentCNIVersion = "2018.08.0"
+	CNIGitHash = "a134a973585b560439ed25ec3857e4789bfeb89f"
```

Comment:
does this need to be public?

Reply:
Yeah, this variable does not need to be public.